setup.py

```diff
@@ -1,9 +1,21 @@
 import setuptools
 
 
+with open("README.md", "r", encoding="utf-8") as desc:
+    long_description = desc.read()
+
+
 setuptools.setup(
     name="xtractmime",
-    version="0.0.0",
+    version="1.0.0",
```

Comment:
```suggestion
    version="0.0.0",
```

This version should be modified in tandem with the one in .bumpversion.cfg (ideally by using the `bumpversion` CLI tool, but it's technically not mandatory). Let's hold on incrementing it until we make a release.